spec/models/item_spec.rb

```diff
@@ -64,12 +64,12 @@
     expect(item2).to_not be_valid
   end
 
-  xit 'must belong to at least one category' do
+  it 'must belong to at least one category' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.